wrappers/wrapper.rb

```diff
@@ -478,6 +478,7 @@ def add_unassigned(unfeasible, vrp, service, reason)
 
             add_unassigned(unfeasible, vrp, service_in, "In a relation with an unfeasible service: #{service.id}")
           }
+          vrp.relations.delete(relation)
```

Comment:
If all services of linked_services of a relation are already marked infeasible and removed from the vrp.services list, do we really need to delete the relation? This is related to my last question, what happens to this relation becomes feasible after we merge the sub-problems with another sub-problem? Can't we get away without deleting/modifying the relations?

Reply:
Decided to handle the relation elimination in `as_json` so that if we optimize the dumped vrp (`vrp.as_json`),  it will always give the same solution.